### ISSUE TITLE
Batch renaming causing files to go missing permanently

### ISSUE THREAD

**Issue by rero100** [SimpleMobileTools/Simple-Gallery#1870]
I renamed multiple files using the '%i' pattern and after a few seconds about half of the selected files went missing. I attemped the same thing other two times and in both attempts multiple files were gone from the folder.

**Comment by tibbi** [comment]
what app version and pattern were you using?

**Comment by rero100** [comment]
version 6.15.1 pro. i used the pattern "2020 (%i)"

**Comment by tibbi** [comment]
if you check the folder with some file manager, isnt the issue only a missing file extension? So that the files arent recognized as images

**Comment by rero100** [comment]
Nope. Checked it with two different file managers and the only files in the folder are the images that weren't "deleted" by the bug.

**Comment by tibbi** [comment]
is the folder on an SD card, or internal storage?

**Comment by rero100** [comment]
internal storage.

**Comment by tibbi** [comment]
I cant really reproduce it, nor see what would be the issue there. Can you tell the exact steps? What are the original file names?

**Comment by tibbi** [comment]
shouldnt happen anymore from 6.23.0
